### ISSUE TITLE
Remote - Button: All

### ISSUE THREAD

**Issue by 4tetra4** [soler2000/com.lightwaverf#20]
Not so important but noticed one thing: In the flow the Channel x Button: **All** is not working

**Comment by soler2000** [comment]
I presume your talking about the handheld remote.  When you say all is not working exactly what do you mean,  is that it does not turn off all or that it does not send any signal.  It is configured in the same way as the other buttons,  and could be used to turn on or off an individual device,  currently it is not configured to turn off all the devices associated with the control.

**Comment by 4tetra4** [comment]
Yes it is the remote control.Apology I forgot to mention that this is about the flow editor trigger. This one does not work: 
![remote all button](https://cloud.githubusercontent.com/assets/17168995/15363855/27187bd8-1d12-11e6-8397-d8dbe50eda4a.png)

**Comment by soler2000** [comment]
Agree this is a bug i will look into

**Comment by Swiftnesses** [comment]
Is this fixed? Can we please have insight data?

**Comment by FrankBakermans** [comment]
Hi Soler2000,

I noticed that this bug is still alive, any idea when you are going to smack it?

Cheers,
Frank
